ema/nextjs/src/components/layout/navigation.js

```diff
@@ -1,15 +1,24 @@
 import Link from 'next/link';
+import { useSearchParams } from "next/navigation";
 
 function Navigation({ items, className }) {
+    const searchParams = useSearchParams();
+
     return (
         <nav className={className}>
             <ul className="flex space-x-4 text-white">
                 <li>
-                    <Link href="/">Home</Link>
+                    <Link href={{
+                        pathname: '/',
+                        query: Object.fromEntries(searchParams.entries())
+                    }}>Home</Link>
```

Comment:
Add comments here why were doing this as well.

Reply:
done